docs/boogie-code.md

```diff
@@ -38,6 +38,29 @@ void assume(bool v) {
 }
 ````
 SMACK interprets the `@` symbol by inserting the value of the C variable `v`.
+Since the arguments of variadic functions may be promoted, SMACK allows users
+to append a format character to the `@` symbol as an indicator that the argument,
+rather than its promoted form, should be used in the format string. There are four
+format characters: `'c'`, `'h'`, `'i'`, and `'f'`, which represent that the
```

Comment:
This should be updated to match the changes @keram88 made below to the main switch-case.

Reply:
Done.